docs/tutorial.rst

```diff
@@ -98,17 +98,6 @@ If you want to use our example project, fork our CumulusCI-Test repo:
 
 If you are using the CumulusCI-Test repo with a Developer Edition Salesforce org, you will need to enable Chatter in the org if it is not already enabled.  With Salesforce DX Scratch Orgs, this is handled for you.
 
-Keychain Key
```

Comment:
Do we want to mention the `$CUMULUSCI_KEY` as a potential workaround for people on platforms not as seamlessly supported by `keyring` (headless linux)?

Reply:
Hmm. I think we should put that somewhere, but I don't think the tutorial is the right place unless we can tell people how to know they need to do this.

Reply:
Yeah, a troubleshooting/FAQ page would be a better place for it. If we do want to include it here, we can just start with something like:

> If you see `<KeychainKeyNotFound traceback>` you'll need to `<set env_var/install deps on linux>`

I think the updated exception below might suffice for now, though.